modules/serverless-kafka-broker-tls-default-config.adoc

```diff
@@ -0,0 +1,91 @@
+// Module is included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kafka.adoc
+
+[id="serverless-kafka-broker-tls-default-config_{context}"]
+= Configuring TLS authentication for the Kafka Broker
```

Comment:
And this one has the exact same title as https://github.com/openshift/openshift-docs/pull/39408/files#diff-a861e7b5c742b621d6f5762a7b8d003a89d1d935de452e54dd6d8c1dd10fc282R6
Let's make it clear how they differ in the title.